sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java

```diff
@@ -318,6 +317,11 @@ public Mono<Void> regenerateKeyAsync(KeyKind keyKind) {
                 new DatabaseAccountRegenerateKeyParameters().withKeyKind(keyKind));
     }
 
+    @Override
+    public boolean localAuthDisabled() {
+        return this.innerModel().disableLocalAuth();
```

Comment:
Is `innerModel().disableLocalAuth()` a boolean or Boolean?

Reply:
`innerModel().disableLocalAuth()` is a Boolean.

Reply:
Then this line is wrong. Boolean could be `null`.

Check the other places, there should be a util function.

Reply:
fixed by [commit#513fb17](https://github.com/Azure/azure-sdk-for-java/pull/45273/commits/513fb178924a3f2c1bb1ab8d0e3fe80dbc79edc3)

Reply:
Check `ResourceManagerUtils.toPrimitiveBoolean`

Reply:
`Boolean.TRUE.equals` changed to `ResourceManagerUtils.toPrimitiveBoolean`.